docs/cluster.md

```diff
@@ -5,12 +5,13 @@ The `amp cluster` command manages all cluster-related operations for AMP.
 ```
     $ amp cluster
 
-    Usage:	amp cluster [OPTIONS] COMMAND [ARGS...]
+    Usage:	amp cluster COMMAND
 
     Cluster management operations
 
     Options:
-          --help   Print usage
+      -h, --help            Print usage
+      -s, --server string   Specify server (host:port)
```

Comment:
Actually, `-s` is a root command options that should only be displayed when using `amp --help` and not be repeated on all commands.